.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,31 @@
+<!--
+Please make sure your issue title matchs the Go convention; a summary
+of the problem, prefixed by the primary affected package.
+
+If the issue is a question about usage or similar, please ask at the
+Gonum discussion list rather than filing an issue:
+
+https://groups.google.com/forum/#!forum/gonum-dev
+-->
+### What are you trying to do?
+
+
+### What did you do?
+<!-- Please include a link to a minimal reproducer here. -->
+
+
+### What did you expect to happen?
+
+
+### What actually happened?
+
+
+### What version of Go, Gonum and C implementation are you using?
+<!--
+Paste the output of `go version` and if you are installing Gonum from source, paste
+the output of `(cd $(go env GOPATH)/src/gonum.org/v1/gonum && git rev-parse HEAD)`.
```

Comment:
Perhaps add `$CC --version` then?

Reply:
The issue is much more likely to be a CBLAS/LAPACKE bug than a compiler problem. So I think we can just ask if we need the information.